coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -303,10 +304,7 @@ def process_message(self, message: fedora_messaging.api.Message):
         self.process_lockfiles(commit)
 
     @catch_exceptions_and_continue
-    def process_lockfiles(self, commit=None):
-
-        rev = commit or self.github_repo_branch[len("refs/heads/"):]
```

Comment:
I think this parsing of the branch is important because when building the `raw.githubusercontent.com` URL below we use `{rev}` and using `refs/heads/testing-devel` here isn't going to work IIUC.

- https://raw.githubusercontent.com/coreos/fedora-coreos-config/testing-devel/manifest-lock.x86_64.json

works

- https://raw.githubusercontent.com/coreos/fedora-coreos-config/refs/heads/testing-devel/manifest-lock.x86_64.json

doesn't

Reply:
Good catch, thanks. Fixed!